core/src/main/java/nu/yona/server/subscriptions/entities/BuddyConnectRequestMessage.java

```diff
@@ -28,8 +25,8 @@ public BuddyConnectRequestMessage()
 		super();
 	}
 
-	private BuddyConnectRequestMessage(UUID id, UUID userID, UUID userAnonymizedID, Set<UUID> goalIDs, String nickname,
-			String message, UUID buddyID, boolean isRequestingSending, boolean isRequestingReceiving)
+	private BuddyConnectRequestMessage(UUID id, UUID userID, UUID userAnonymizedID, String nickname, String message, UUID buddyID,
+			boolean isRequestingSending, boolean isRequestingReceiving)
```

Comment:
Good cleanup. In the past, the buddy connect request communicated the set of goals for the user. Nowadays these are simply part of the user anonymized.